src/components/SMSCodeInput.tsx

```diff
@@ -145,11 +146,10 @@ const SMSCodeInput = ({ submitRef, disabled }: { submitRef: RefObject<HTMLButton
 		<fieldset className="flex items-center justify-center space-x-3">
 			<legend className="sr-only">Enter your SMS code</legend>
 			{inputsRefs.map((ref, i) => (
-				<motion.input
+				<input
```

Comment:
@denis-ottofeller Also the SMS-code input goes beyond the boundaries of the display

<details>
<summary>Screenshot</summary>


![image](https://user-images.githubusercontent.com/89008845/206488842-eb224c08-4cb5-422b-9d74-a1a4dce3f3a3.png)
</details>